pyodesys/results.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, xout, yout, params, info, odesys):
         self.params = params
         self.info = info
         self.odesys = odesys
+        self.names = getattr(self.odesys,'names')
```

Comment:
Why the use of getattr versus just `self.odesys.names`?

Reply:
Because I haven't explored the documentation of the other class in enough depth. :^(